front/app/api/events/useEventsByUserId.ts

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import eventsKeys from './keys';
+import { IEvents, EventsKeys, InputParameters } from './types';
+
+const fetchEvents = ({ attendeeId }: InputParameters) => {
```

Comment:
Minor. We could use a parameter and simplify the logic on line 15. We could ensure that the call is not made if the `attendeeId` is missing. That is if that is a possibility
```suggestion
const fetchEvents = (attendeeId: string) => {
```